distribution/std-lib/Base/src/Data/Any/Extensions.enso

```diff
@@ -68,6 +68,19 @@ Any.is_nothing = case this of
 Any.catch : (Error -> Any) -> Any
 Any.catch (handler = x->x) = this.catch_primitive handler
 
+## Maps a dataflow error.
+   If the original value was a non-error value, it is not affected, but if it
+   was an error, the error is mapped through the provided function.
+
+   Arguments:
+   - f: The function to transform the error.
+
+   > Example
+     Wrapping an error value.
+       map.get "x" . map_error (_ -> ElementNotFound "x")
```

Comment:
Since you're already here, could you also add `is_error` please?

Reply:
Sure, but I'm not sure what the correct implementation would be for that.

Would the following be correct?
```
is_error =
    non_error = case this of
        _ -> False
    non_error.catch (_ -> True)
```